modules/util/lang-builder/src/main/java/com/liferay/lang/builder/LangBuilder.java

```diff
@@ -187,6 +187,7 @@ public LangBuilder(
 		_createProperties(content, "ja"); // Japanese
 		_createProperties(content, "kk"); // Kazakh
 		_createProperties(content, "ko"); // Korean
+		_createProperties(content, "km"); // Cambodian
```

Comment:
One more minor thing, can you help move this line up 1 line? so adding `km` should go before `ko`. Thanks!